driver-core/src/test/java/com/datastax/driver/core/CCMBridge.java

```diff
@@ -486,6 +487,7 @@ private void maybeInitCluster(){
                 try {
                     //launch ccm cluster
                     ccmBridge = CCMBridge.create("test-class");
+                    ccmBridge.updateConfig("enable_user_defined_functions", "true");
```

Comment:
What if the tests are being run against an older version of Cassandra where the flag is not defined?